### PR TITLE
FS-1355-assessment-progress-percentage

### PR DESCRIPTION
**Pull request by ramsharma-prog**
Pull request to retrieve the assessment progress for each application and display it on the assessment dashboard.

### CODE REVIEW

File: config/envs/default.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ class DefaultConfig:
     ASSESSMENT_SCORES_ENDPOINT = ASSESSMENT_STORE_API_HOST + "/score"
 
     ASSESSMENT_COMMENT_ENDPOINT = ASSESSMENT_STORE_API_HOST + "/comment"
+    ASSESSMENT_PROGRESS = ASSESSMENT_STORE_API_HOST + "/bulk-progress"
```
Comment (srh-sloan) [inline config/envs/default.py:106]: Can we call this `ASSESSMENT_PROGRESS_ENDPOINT` pls?

File: app/assess/data.py [hunk 2]
```diff
@@ -47,6 +47,21 @@ def get_data(
...
+        'application_ids':
+            [x.get('application_id') for x in application_metadata]
+        }
+    endpoint_url = Config.ASSESSMENT_PROGRESS
+    response = get_data(endpoint=endpoint_url, payload=application_ids_list)
+    if response is not None:
+        [x.update(
+            {"progress":res.get("progress")}
+            ) for res in response for x in application_metadata
+            if res['application_id']==x['application_id']]
```
Comment (srh-sloan) [inline app/assess/data.py:61]: Did you run the pre-commit checks? Indenting looks a bit odd?

**Review by srh-sloan — CHANGES_REQUESTED** [review]
Just a couple minor comments :)

File: app/assess/data.py (hunk 2 above)
Comment (ramsharma-prog) [reply app/assess/data.py:61]: Haha no I thought I will run at the end as quite few lines looks add on the file so I didn't want to confuse it 😄

File: config/envs/default.py (hunk 1 above)
Comment (ramsharma-prog) [reply config/envs/default.py:106]: Will do 👍

**Review by gio-karunakaran — APPROVED** [review]
LGTM

File: app/assess/data.py (hunk 2 above)
Comment (ramsharma-prog) [reply app/assess/data.py:61]: Done!

File: config/envs/default.py (hunk 1 above)
Comment (ramsharma-prog) [reply config/envs/default.py:106]: Done!

**Review by srh-sloan — APPROVED** [review]
Good to go 👍

**Review by johnoneill46 — APPROVED** [review]

**Pull request merged**